a08.Evaluator/Evaluator.cs

```diff
@@ -38,11 +42,11 @@ class Evaluator {
 
    void Process (Token token) {
       switch (token) {
-         case TNumber num: 
-            mOperands.Push (num.Value); 
+         case TNumber num:
+            mOperands.Push (num.Value);

```

Comment:
Why 'mOperands', 'mOperators' are readonly? And declare the private variables at the end of the program.